src/khoj/routers/indexer.py

```diff
@@ -58,7 +57,7 @@ class IndexerInput(BaseModel):
 
 @indexer.post("/batch")
 async def index_batch(
-    request: Request,
```

Comment:
Could you add telemetry here as well? We'll need to retain the `request` object in that case.

Reply:
Added telemetry in e347823